Automattic/jetpack
# RNA Connection: rename Main component into ConnectButton

#### Changes proposed in this Pull Request:
- Rename the `Main` component into `ConnectButton`

#### Jetpack product discussion
1200295095943283-as-1200465631534663

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Disconnect Jetpack.
2. Go to "Tools -> Connection Manager".
3. Click "Connect", you should be taken to the Calypso flow.
4. Complete connection, confirm that the page displays "Site and User Connected".
